src/distributions/normal.jl

```diff
@@ -538,13 +538,17 @@ end
 
 # Thes functions extends the `CVI` approximation method in case if input is from the `NormalDistributionsFamily`
 
-get_df_m(::Type{<:UnivariateNormalNaturalParameters}, ::Type{<:UnivariateGaussianDistributionsFamily}, logp_nc::Function) = (z) -> ForwardDiff.derivative(logp_nc, z)
+get_df_m(::ForwardDiffGrad, ::Type{<:UnivariateNormalNaturalParameters}, ::Type{<:UnivariateGaussianDistributionsFamily}, logp_nc::Function) =
+    (z) -> ForwardDiff.derivative(logp_nc, z)
 
-get_df_m(::Type{<:MultivariateNormalNaturalParameters}, ::Type{<:MultivariateGaussianDistributionsFamily}, logp_nc::Function) = (z) -> ForwardDiff.gradient(logp_nc, z)
+get_df_m(::ForwardDiffGrad, ::Type{<:MultivariateNormalNaturalParameters}, ::Type{<:MultivariateGaussianDistributionsFamily}, logp_nc::Function) =
+    (z) -> ForwardDiff.gradient(logp_nc, z)
 
-get_df_v(::Type{<:UnivariateNormalNaturalParameters}, ::Type{<:UnivariateGaussianDistributionsFamily}, df_m::Function) = (z) -> ForwardDiff.derivative(df_m, z)
+get_df_v(::ForwardDiffGrad, ::Type{<:UnivariateNormalNaturalParameters}, ::Type{<:UnivariateGaussianDistributionsFamily}, logp_nc::Function, df_m::Function) =
+    (z) -> ForwardDiff.derivative(df_m, z)
 
-get_df_v(::Type{<:MultivariateNormalNaturalParameters}, ::Type{<:MultivariateGaussianDistributionsFamily}, df_m::Function) = (z) -> ForwardDiff.jacobian(df_m, z)
+get_df_v(::ForwardDiffGrad, ::Type{<:MultivariateNormalNaturalParameters}, ::Type{<:MultivariateGaussianDistributionsFamily}, logp_nc::Function, df_m::Function) =
+    (z) -> ForwardDiff.jacobian(df_m, z)
```

Comment:
Can we get rid of these functions if we call `compute_grad()` or `compute_hessian` in the function body of `get_df_m` etc in `ReactiveMP.jl`? Let's discuss

Reply:
We get rid of [these functions](https://github.com/biaslab/ReactiveMP.jl/blob/cvi-stability-fix/src/distributions/normal.jl#L541-L556). Right now, we are using only `compute_gradient`, `compute_hessian`, `compute_derivative`.